queries/r/textobjects.scm

```diff
@@ -79,44 +67,44 @@
   .
   (_) @parameter.inner
   .
-  ","? @_end)
+  (comma)? @_end)
   (#make-range! "parameter.outer" @parameter.inner @_end))
 
 ; number
 (float) @number.inner
 
 ; assignment
-(left_assignment
+("<-"
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(left_assignment
+("<-"
   name: (_) @assignment.inner)
 
-(right_assignment
+("->"
   value: (_) @assignment.inner @assignment.lhs
   name: (_) @assignment.rhs) @assignment.outer
 
-(right_assignment
+("->"
   name: (_) @assignment.inner)
 
-(equals_assignment
+((comma)
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(equals_assignment
+((comma)
   name: (_) @assignment.inner)
 
-(super_assignment
+("<<-"
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(super_assignment
+("<<-"
   name: (_) @assignment.inner)
 
-(super_right_assignment
+("->>"
   value: (_) @assignment.inner @assignment.lhs
   name: (_) @assignment.rhs) @assignment.outer
 
-(super_right_assignment
-  name: (_) @assignment.inner)
+(binary_operator)
+  name: (_) @assignment.inner
```

Comment:
should this be

```lisp
(binary_operator
   name: (_) @assignment.inner)
```

? I am not super familiar with treesitter query grammer. Forgive me if I am actually wrong.

Reply:
I think this is ok like this. If I try your suggestion, I have an error.

Reply:
> I think this is ok like this. If I try your suggestion, I have an error.

Yes, tried and see the error. Just ignore my suggestion.